### ISSUE TITLE
Lua for scripting and settings files.

### ISSUE THREAD

**Issue by Newlifer** [codingteam/lambda-ray#5]
Yes, scripting isn't so necessary for now, but to use lua to make good and flexible config files would be helpful.

**Comment by ForNeVeR** [comment]
Are you guys sure that should be lua? Why not YAML, for example?

**Comment by hagane** [comment]
I second this question. We've not even established what parameters we want to expose through configuration files yet.

**Comment by Newlifer** [comment]
Yes, sure, we can use yaml or something. It's just to discuss for future, not to get solution right now. (=
